package.json

```diff
@@ -12,21 +12,25 @@
     "typecheck": "tsc --project tsconfig.build.json --noEmit",
     "test": "vitest run --globals",
     "test:dev": "vitest --globals",
+    "test:jest": "jest --config=jest.config.js",
     "release": "npm run build && npm version patch && npm publish"
   },
   "author": "Ridestore",
   "license": "ISC",
   "devDependencies": {
     "@managed-components/types": "^1.0.4",
+    "@types/jest": "^29.4.0",
     "@typescript-eslint/eslint-plugin": "^5.27.0",
     "all-contributors-cli": "^6.20.0",
     "esbuild": "^0.14.42",
     "eslint": "^8.16.0",
     "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-prettier": "^4.0.0",
+    "jest": "^29.5.0",
```

Comment:
Just spreading the love in case you weren't already aware: https://vitest.dev/ is a drop-in replacement for jest that is delightfully fast, and typescript ready out of the box.

Reply:
thanks for sharing!